common/Mesh-Network-Joining-Process-Comparison.md

```diff
@@ -0,0 +1,540 @@
+
+## 3.4. Provisioning behavior
+The process which transforms an unprovisioned device into a node is called "provisioning". Consider purchasing a new Bluetooth light with mesh support, bringing it home and setting it up. To make it part of your mesh network, so that it can be controlled by your existing Bluetooth light switches and dimmers, you would need to provision it.   
+
+Provisioning is a secure procedure which results in an unprovisioned device possessing a series of encryption keys and being known to the Provisioner device, typically a tablet or smartphone. One of these keys is called the network key. All nodes in a mesh network possess at least one NetKey and it is possession of this key which makes a device a member of the corresponding network and as such, a node.    
+
+Provisioning is performed using a five-step process: beaconing, invitation, exchanging public keys, authentication, and distribution of the provisioning data.   
+
+### 3.4.1. Beaconing
+An unprovisioned device may support PB-ADV and may support PB-GATT. It is strongly recommended to support PB-ADV and PB-GATT in Bluetooth Mesh profile specification. And we will focus on the PB-ADV since most of the unprovisioned device support it.   
+A Provisioner shall support at least one of PB-ADV or PB-GATT. It is strongly recommended to support PB-ADV in Bluetooth Mesh profile, however, most of the smart phone do support that at this point.   
+
+#### 3.4.1.1. PB-ADV
+PB-ADV is a provisioning bearer used to provision a device using *Generic Provisioning PDUs* over the advertising channels. A device that supports PB-ADV, has not been provisioned, and is not in the process of being provisioned, shall advertise the *Unprovisioned Device beacon*, otherwise a device shall not advertise the Unprovisioned Device beacon.   
+
+The Unprovisioned Device beacon is used by devices that are unprovisioned to allow them to be discovered by a Provisioner.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-unprovisioned-device-beacon-format.png">  
+</div>  
+<div align="center">
+  <b>  Unprovisioned device beacon format </b>
+</div>  
+<br>
+
+The out-of-band (OOB) Information field in the Unprovisioned Device beacon is used to help drive the provisioning process by indicating the availability of OOB data. A separate non-connectable advertising packet with a ***Uniform Resource Identifer*** (URI) data type that points to OOB information will be advertised if use OOB. In this case, to allow the association of the advertised URI with the Unprovisioned Device beacon, the beacon may contain an optional 4-octet URI Hash field.   
+
+Below is a example of the Unprovisioned Device beacon of the device that supports PB-ADV, the Device UUID is ```{0x53,0x69,0x6C,0x61,0x62,0x73,0x44,0x65,0x76,0x2D,0x95,0x8D,0x64,0x57,0x0B,0x00}``` and it doesn't use the OOB, so the URI Hash field is not included there.   
```

Comment:
[Mark] I haven't had a chance to look at rest part of section 3. Maybe Eric can put more comments on this.